### DISCUSSION TITLE
How to show custom dialog when uninstall?

### DISCUSSION THREAD

**Discussion by truecode112** [wixtoolset/issues#7637] [Questions]
Hello everyone
In my product, I want to show custom UI dialog when uninstall.
I found that MSI uninstaller works silently.
How to hook this and show custom dialog when uninstall product?
Anyone knows how, please help me
Regards

**Comment by robmen** [comment]
The Windows Installer doesn't show full UI during uninstall. You can have a lot more control using a Bundle.
